Makefile

```diff
@@ -27,4 +31,18 @@ test:
 test-coverage:
 	py.test --verbose --cov-report term --cov=integrade --cov=tests tests
 
+ci-oc-login:
+	./oc login $(OPENSHIFT_URL) --token=$(OPENSHIFT_TOKEN)
+
+ci-refresh-cloudigrade:
+	REVISION=$$(./oc rollout latest dc/postgresql -o revision)
+	./oc rollout status dc/postgresql --revision=$(REVISION)
+	REVISION=$$(./oc rollout latest dc/cloudigrade -o revision)
+	./oc rollout status dc/cloudigrade --revision=$(REVISION)
```

Comment:
looks like maybe something funny is going on with setting this env variable from the most recent travis build

Reply:
After a lot of experimentation I've decided to rollback the changes to the makefile. It simply doesn't handle env var assignment and command execution in a predictable manner to make it useful.